pydomcfg/tests/bathymetry.py

```diff
@@ -69,17 +117,53 @@ def _add_attributes(ds: Dataset) -> Dataset:
     Parameters
     ----------
     ds: Dataset
-        Dataset with bathymetry and mask variables
+        Dataset with bathymetry and mask variables [and rmax]
 
     Returns
     -------
     Dataset
     """
+
     attrs_dict = {
         "Bathymetry": dict(standard_name="sea_floor_depth_below_geoid", units="m"),
         "mask": dict(standard_name="sea_binary_mask", units="1"),
     }
+
+    if "rmax" in list(ds.keys()):
```

Comment:
```suggestion
    if "rmax" in ds:
```

Reply:
ok. I do my coding in Spyder and it was throwing me `Output from spyder call 'get_namespace_view':` when I had originally coded `if "rmax" in ds:` yesterday, that's while I thought I'd try and clean the std_out by explicitly listing keys with `if "rmax" in list(ds.keys()):`. Outside of Spyder I don't get any addition messages to std_out, so I'll implement  `if "rmax" in ds:`

Reply:
Ohh, never used spyder, not sure what that message means... BTW, you can tread `ds` like a dictionary, so this also works `list(ds)`